### PR TITLE
Haris/dotnet email verify sms

### PR DESCRIPTION
**Pull request by Haris-mslm**

### CODE REVIEW

**Comment by Haris-mslm** [comment]
@asadmahasg @UmanShahzad

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 1.0.0
+
+- Initial release of Email Verify and Otp SDK.
```
Comment (UmanShahzad) [inline CHANGELOG.md:5]:
```suggestion
- Initial release of Email Verify and OTP SDK.
```

File: Mslm.Tests/EmailVerifyTests.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace Mslm.Tests
+{
+    public class EmailVerifyTests
+    {
+        [Fact]
+        public async Task TestSingleVerify()
+        {
+            // Arrange
+            // to run this test, add your MSLM Api Key to environment variable
```
Comment (UmanShahzad) [inline Mslm.Tests/EmailVerifyTests.cs:13]: `Mslm` not `MSLM` (in all places when it's written in prose)

API, however should be all-caps, not `Api` when written in prose

File: README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+```cs
+using System;
+using Mslm.MslmNS;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+		// Initialize Mslm Client.
```
Comment (UmanShahzad) [inline README.md:95]: Accidental tab

File: README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        Console.WriteLine($"Code: {response.Code}");
+        Console.WriteLine($"Msg: {response.Msg}");
+    }
+}
+
+```
+
+## About Mslm
+
+mslm focuses on producing world-class business solutions. It’s the
```
Comment (UmanShahzad) [inline README.md:137]: Capital M

File: src/Mslm/Mslm/Mslm.cs [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    /// <summary>
+    /// Represents the main client for the Mslm SDK, providing access to Email Verification and OTP services.
+    /// </summary>
+    public class MslmClient
+    {
+        // Common util class.
+        public Lib LibClient { get; private set; }
+
+        // The Email Verify API client.
+        public EmailVerify EmailVerifyClient { get; private set; }
```
Comment (UmanShahzad) [inline src/Mslm/Mslm/Mslm.cs:16]: Not sure if this'll cause a name conflict but the variable name itself should be just `EmailVerify` so it's easier to reach into it when a user uses the global Mslm client

Comment (Haris-mslm) [reply src/Mslm/Mslm/Mslm.cs:16]: It actually allows but c# gives a warning and can cause conflicts in some cases therefore this is generally not considered a good practice to have types and variables with same names. Same goes for namespaces and classes. But I'll change it for now.

**Pull request merged**